frontend/app/src/components/home/DeletedContent.svelte

```diff
@@ -16,9 +17,14 @@
 </script>
 
 <div class="deleted">
-    {$_("messageDeleted", {
-        values: { username, timestamp: timestampStr },
-    })}
+    <p>
+        {$_("messageDeleted", {
+            values: { username, timestamp: timestampStr },
+        })}    
+    </p>
+    {#if undeleting}
```

Comment:
isn't this going to cause an unpleasant jiggle (since it adds height). I had in mind that we would just _replace_ the original message with an "undeleting message ..." placeholder.

Reply:
Yes I did start doing that and that was also a bit weird.